src/main/java/dev/snowdrop/release/App.java

```diff
@@ -30,6 +28,10 @@
 public class App implements QuarkusApplication {
     public final static String CVE_REPORT_REPO_NAME = "snowdrop/reports";
 
+    @Inject
+    @ConfigProperty(name = "gitlab.buildconfig.fork.repo", defaultValue = "snowdrop/build-configurations")
+    String buildConfigFormRepoName;
```

Comment:
Why `buildConfigFormRepoName`, is the `form` correct? , Did you mean `fork` with `k`?

Reply:
Yes, you're right.

Reply:
Fixed